src/worker/activities/local_activities.rs

```diff
@@ -170,23 +212,19 @@ impl LocalActivityManager {
                             runtime: Duration::from_secs(0),
                             attempt: 0,
                             backoff: None,
+                            original_schedule_time: None,
```

Comment:
Why `None`?

Reply:
There's no meaningful value to put here at the moment - and it will never be used anyway since we've cancelled.